torch/onnx/verification.py

```diff
@@ -3,18 +3,22 @@
 ONNX Runtime is required, and is used as the ONNX backend for export verification.
 """
 
+
 import contextlib
 import copy
+import difflib
 import io
 import os
 import tempfile
 import warnings
-from typing import Any, Mapping, Optional, Sequence, Tuple, Union
+from typing import Any, Mapping, Optional, Sequence, Set, Tuple, Type, Union
 
 import numpy as np
 
 import torch
-from torch import Tensor
+import torch._C._onnx as _C_onnx
+from torch import _C, Tensor
```

Comment:
Use `torch.Tensor`

Reply:
No strong opinion. `import Tensor` appears to be a convention in PyTorch. Perhaps due to its usage mainly in type annotations, hence following the typings import style. I'm changing to `torch.Tensor` now. As long as we are consistent I'm ok.